lib/rspec/rails/fixture_file_upload_support.rb

```diff
@@ -0,0 +1,38 @@
+require 'singleton'
```

Comment:
This can't be used as it would pollute any codebase using 'rspec-rails', please refactor to not need this.